acl-lib/src/cmr/acl/core.clj

```diff
@@ -28,11 +28,18 @@
 (defn get-token
   "Returns the token the user passed in the headers or parameters"
   [params headers]
-  (let [non-empty-string #(when-not (str/blank? %) %)]
-    (or (non-empty-string (get headers tc/token-header))
-        (non-empty-string (:token params))
-        (when (allow-echo-token) 
-          (non-empty-string (get headers tc/echo-token-header))))))
+  (let [non-empty-string #(when-not (str/blank? %) %)
```

Comment:
nit: Make `non-empty-string` a function rather than define it in the `let` block.